.taskcluster.yml

```diff
@@ -152,20 +152,18 @@ tasks:
                 routes:
                     $flattenDeep:
                         - checks
-                        - $if: 'level == "3"'
+                        - tc-treeherder.v2.${project}.${head_sha}
```

Comment:
@JohanLorenzo I was looking at the performance-tests bitrise task with @jjSDET and noticed that this route was missing from it for some reason and it's the route we need to get the logs ingested by treeherder: https://firefox-ci-tc.services.mozilla.com/tasks/LWDiMLyxQzur7cuc4TjFzA

I'm not sure why it's not working though because it looks like the performance-tests task is level-3?

Reply:
Ah, I see what's going on. This line only defines the route of the decision task[1]:

<img width="939" alt="image" src="https://user-images.githubusercontent.com/5907366/234228800-50194ed4-8a69-40c5-bb86-a4f683c8768d.png">

We have to add a treeherder[2] block like this one to the `kind.yml`, then taskgraph will take care of setting the right route for us[3].

Let me know if I can provide additional information 🙂 

[1] https://firefox-ci-tc.services.mozilla.com/tasks/MGIrBQ2yTJOctT36x4PkKQ
[2] https://github.com/mozilla-mobile/firefox-android/blob/825b3aafeccdbfd6306521b68bb405f68f45d287/taskcluster/ci/startup-test/kind.yml#L13
[3] https://github.com/taskcluster/taskgraph/blob/d339ac59186fc24fa8afb4162ada1b5fac44d2a9/src/taskgraph/transforms/task.py#L1004